### DISCUSSION TITLE
Auth Rocket - Configuration

### DISCUSSION THREAD

**Discussion by LaiaPerez88** [boostercloud/booster#945] [Q&A]
David Muczynski asked us this via email:

Was going to demo this to a client but I hit a wall. Documentation/Implementation question. 
I see this in the documentation but it fails. AWS not used, Provider not declared, etc... What does this need to be to actually use congnito authentication. Do you have any example repos available? Thanks import { Booster } from '@boostercloud/framework-core' import { BoosterConfig } from '@boostercloud/framework-types' import * as AWS from '@boostercloud/framework-provider-aws' Booster.configure('production', (config: BoosterConfig): void => { config.appName = 'my-store' config.provider = Provider([ { packageName: '@boostercloud/rocket-auth-aws-infrastructure', parameters: { mode: 'Passwordless' }, }, ]) })

**Comment by NickSeagull** [comment]
Auth rocket has been deprecated, closing
